UDPClientGUI/SiceUDPClient.cs

```diff
@@ -62,22 +63,29 @@ public void Dispose()
 
         public void Handle(ClientCommandEvent Command)
         {
+            Dictionary<ClientCommandEvent.Command, System.Action> handler = new Dictionary<ClientCommandEvent.Command, System.Action>();
+            handler.Add(ClientCommandEvent.Command.Connect, Connect);
+            handler.Add(ClientCommandEvent.Command.Disconnect, Disconnect);
+            handler.Add(ClientCommandEvent.Command.SendMessage, SendMessage);
```

Comment:
This should be initialised in the constructor, but don't worry about it for now. 

Otherwise there's no advantage in having it.